doc/source/releases/_thanks.rst

```diff
@@ -1,4 +1,4 @@
-To stay up to date with Galaxy's progress watch our `screencasts <https://vimeo.com/galaxyproject>`__,
+To stay up to date with Galaxy's progress watch our `screencasts <https://www.youtube.com/c/galaxyproject>`__,
```

Comment:
@nsoranzo This .rst is okay?

Reply:
It is! It's included in all release note pages, so it makes sense to update.